opium/src/response.ml

```diff
@@ -161,6 +161,23 @@ let of_json ?version ?status ?reason ?headers ?env body =
     (body |> Yojson.Safe.to_string)
 ;;
 
+let of_file ?version ?reason ?headers ?env fname =
```

Comment:
This could take an optional `mime` argument that would override the detected one.

Reply:
I was thinking that this could be provided as header but it probably is a quite common use case, fixed.